.yarn/patches/fsevents-npm-1.2.13-9e970219e7.patch

```diff
@@ -0,0 +1,12 @@
+diff --git a/fsevents.js b/fsevents.js
+index 939cc7e3d177dd2d4f889f8e97a5b7d3c59fc68f..e469059ac6defb501e860e549faefc306ec12bac 100644
+--- a/fsevents.js
++++ b/fsevents.js
+@@ -73,6 +73,7 @@ function proxies(ctor, target) {
+ function getFileType(flags) {
+   if (Native.Constants.kFSEventStreamEventFlagItemIsFile & flags) return 'file';
+   if (Native.Constants.kFSEventStreamEventFlagItemIsDir & flags) return 'directory';
++  if (Native.Constants.kFSEventStreamEventFlagMustScanSubDirs & flags) return 'directory';
```

Comment:
For more information about this change see this issue: https://github.com/vitejs/vite/issues/14043
The storybook restart issue in local was related to the dependency `fsevents` used in `chodikar` (the file watcher used in storybook). The issue has been fixed in the latest version of `fsevents` but this version seems incompatible on linux atm so we're patching the dependency with the fix in the meantime.